.github/workflows/ghcr.yaml

```diff
@@ -20,6 +20,12 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Set up QEMU
```

Comment:
it this required ? was looking in docs
```
[setup-qemu](https://github.com/docker/setup-qemu-action) action can be useful if you want to add emulation support with QEMU to be able to build against more platforms.
```
I don't have much idea about that 😅 but are we some how be using `emulation support`?

Reply:
@sm43 yes, it is required to build ppc64le artifacts on top of x86_64 hardware. Many OSS have started adopting this to their CI's https://github.com/redhat-openshift-ecosystem/openshift-preflight/blob/main/.github/workflows/build-main.yml#:~:text=%2D%20name%3A%20Set%20up,docker/setup%2Dqemu%2Daction%40v1

Reply:
ah okay.
Thanks for the pr 🙃

Reply:
Thanks :)